index.js

```diff
@@ -46,6 +48,29 @@ export const provideLinter = () => ({
 	lint
 });
 
-export const activate = () => {
+export function activate() {
 	require('atom-package-deps').install();
-};
+	this.subscriptions = new CompositeDisposable();
+	this.subscriptions.add(atom.commands.add('atom-text-editor', {
+		'linter-xo:fix-file': function fixFile() {
+			const textEditor = atom.workspace.getActiveTextEditor();
+			if (!textEditor || textEditor.isModified()) {
+				// Ignore invalid or unsaved text editors
+				return;
+			}
+			const filePath = textEditor.getPath();
+			const fileDir = path.dirname(filePath);
+			new BufferedNodeProcess({
```

Comment:
I've got no strong reason to use the CLI instead of using the API. I just found CLI to work, while the API is still in the works. It uses a `BufferedNodeProcess` which uses the current electron as the runtime and the bundled `xo` as the library.

I'm happy to change this once you implement an API like that